core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -315,12 +312,13 @@ public GetTransactionResponse findTransaction(JwtToken token, GetTransactionRequ
     }
 
     // We should not return the transaction that belongs to other accounts.
-    if (txn == null || !txn.getStellarAccount().equals(token.getAccount())) {
+    if (txn == null || !txn.getSep10Account().equals(token.getAccount())) {
```

Comment:
❓ Which accounts could be used for the SEP-10? I'm assuming it could be a user wallet or a custodial wallet, is that correct?

Reply:
That's correct.